header-gossip-network.md

```diff
@@ -0,0 +1,143 @@
+# Portal Network: Header Gossip
+
+This document is the specification for the "Header Gossip" network which is responsible for dissemination of new headers as new blocks are mined and  transmission of recent snapshots of the "Header Accumulator" data structure which all nodes on the network are expected to maintain.
+
+## Design Requirements
+
+The network functionality has been designed around the following requirements.
+
+- A DHT node can reliably receive the headers for new blocks via the gossip mechanism in a timely manner.
+- A DHT node can retrieve a recent snapshot of another DHT node's "Header Accumulator"
+
+
+## The "Header Accumulator"
+
+The "Header Accumulator" is based on the [double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571) that is currently used in the beacon chain.  This data structure is designed to allow nodes in the network to "forget" the deeper history of the chain, while still being able to reliably receive historical headers with a proof that the received header is indeed from the canonical chain (as opposed to an uncle mined at the same block height).
+
+The accumulator is defined as an [SSZ](https://ssz.dev/) data structure with the following schema:
+
+```python
+EPOCH_SIZE = 8192
```

Comment:
```suggestion
EPOCH_SIZE = 8192 # blocks
```
Perhaps a stupid remark, as it's implied by reading the code, but its nowhere explained what an epoch is.

Reply:
Not stupid at all.  These are the kinds of obvious omissions that get made when you are way too close to the subject matter to adequately zoom out and describe everything thoroughly.